completion/zsh/_tlp-radio-device

```diff
@@ -0,0 +1,6 @@
+#compdef bluetooth nfc wifi wwan
+
+_values 'command' \
+    'on[switch device on]' \
+    'off[switch device off]' \
+    'toggle[toggle device state]'
```

Comment:
When I enter (or complete to)  `wifi on ` (note the trailing space) the completion will continue and happily add any number of _on_ / _off_ / _toggle_ arguments. There should only be one. Also happens for _off_ / _toggle_ as 1st argument and for `nfc`, `bluetooth` and `wwan` as well.

Reply:
same problem as _tlp_rdw. Clearly _values did not quite work how I thought it did.